### PR TITLE
Add OAuth APIs

### PR DESCRIPTION
**Pull request by morugu**
add [OAuth APIs](https://developers.line.me/en/reference/messaging-api/#issue-channel-access-token) feature.

### CODE REVIEW

**Comment by morugu** [comment]
thanks for comments. I'll fix code and update docs.

File: lib/types.ts [hunk 1]
```diff
@@ -1488,3 +1488,19 @@ export type RichMenu = {
...
 
 export type RichMenuResponse = { richMenuId: string } & RichMenu;
+
+export type IssueAccessTokenRequest = {
+  grant_type: "client_credentials";
+  client_id: string;
+  client_secret: string;
+};
+
+export type IssueAccessTokenResponse = {
```
[the remark this answers is not in the document]
Comment (morugu) [reply lib/types.ts]: issue access token's response has used snake case in docs
https://developers.line.me/en/reference/messaging-api/#issue-channel-access-token

is that ok using camel case?

**Comment by Dhome12** [comment]
I can not apply anything please explain how to add an order ( npm install )  Does not respond to the application ( Termux )

**Comment by utatti** [comment]
@Dhome12 

I assume this p-r is not related to your question. Please create a new issue, preferably with more detail.

**Comment by utatti** [comment]
I close this p-r for inactivity.

Please feel free to reopen the p-r with conflicts resolved!

**Comment by tkgauri** [comment]
#142

**Pull request closed without merging**